charts/k8ssandra-cluster/templates/cassdc.yaml

```diff
@@ -22,12 +22,16 @@ spec:
   size: {{ .Values.size }}
   storageConfig:
     cassandraDataVolumeClaimSpec:
+{{- if (eq "-" .Values.k8ssandra.cassandraLibDirVolume.storageClass) }}
```

Comment:
@vmarchese Can you change to instead check if `storageClass` is undefined/unset?

Reply:
It already checks the existence of the value `storageClass`. 
If it is not defined, it defaults to "standard" as before.

So, if the values file is:

```yaml
k8ssandra:
  namespace: default
  name: k8ssandra
...
```

the CR is:
```yaml
apiVersion: cassandra.datastax.com/v1beta1
kind: CassandraDatacenter
metadata:
  name: dc1



Did I get your point ?

Reply:
I interpreted `{{- if (eq "-" .Values.k8ssandra.cassandraLibDirVolume.storageClass) }}` as a check to see if `storageClass` equals `-`. I thought it could something like:

```yaml
{{ .Values.k8ssandra.cassandraLibDirVolume.storageClass | default "standard" }}
```